lib/algos/list/basic.js

```diff
@@ -44,6 +46,50 @@ class List extends Extension {
         return params;
     }
 
+    /**
+     * Filters customAttributes sub-object if present
+     *
+     * @param {String} value - The JSON value of a listing item
+     *
+     * @return {Boolean} Returns false if the value is filtered out, else true.
+     */
+    customFilter(value) {
+        let ret = false;
+        try {
+            const _value = JSON.parse(value);
```

Comment:
I would only try/catch the JSON parse as that is the explicit error we return later on